frontend/components/inbox/TicketOverview.vue

```diff
@@ -21,56 +12,56 @@
         </div>
 
         <submit-button
-                :class="showPersonal ? `bg-orange-500 text-white` : `bg-gray-100 text-black` "
-                @click="togglePersonal"
-                class="px-2 md:m-0 h-10"
-                v-if="is_staff"
+          :class="showPersonal ? `bg-orange-500 text-white` : `bg-gray-100 text-black` "
+          @click="togglePersonal"
+          class="px-2 md:m-0 h-10"
+          v-if="is_staff"
         >
           <font-awesome-icon :icon="showPersonal ? 'check' : 'minus'" class="mr-2"></font-awesome-icon>
           My Tickets
         </submit-button>
-
-        <!-- Change view -->
-        <button
-                class="hidden md:block border rounded h-10 px-3 focus:outline-none hover:bg-gray-100"
-                :title="list ? 'Show Columns View' : 'Show List View'"
-                @click="toggleView"
-        >
-          <font-awesome-icon :icon="list ? 'columns' : 'list'"></font-awesome-icon>
-        </button>
       </div>
     </div>
 
-    <!-- List -->
-    <div v-if="list" class="container mx-auto flex flex-col space-y-4 mb-4">
-      <ticket-list
-              v-for="(column, i) in tickets"
-              :key="i"
-              :color="colors[column.label]"
-              :title="column.label"
-              :personal="showPersonal"
-              :ticket-list="column.tickets"
-              :has_next="column.has_next"
-              :length="column.total"
-              @input="loadStatus(column.label)"
-      />
-    </div>
-
     <!-- Columns -->
-    <div v-else class="max-w-full flex md:space-x-4 flex-grow overflow-x-auto px-4 mb-4 space-x-2">
+    <div class="max-w-full flex flex-grow overflow-x-auto pl-4 b-4">
       <ticket-column
-              v-for="(column, i) in tickets"
-              :key="i"
-              :color="colors[column.label]"
-              :title="column.label"
-              :personal="showPersonal"
-              :ticket-list="column.tickets"
-              :has_next="column.has_next"
-              :length="column.total"
-              @input="loadStatus(column.label)"
-              class="min-w-3/4 sm:min-w-1/2 md:min-w-0"
+        v-for="(column, i) in tickets"
+        :key="i"
```

Comment:
Ik lees op internet dat het gebruiken van een index als key niet optmaal is, miss de unieke ticket id gebruiken?

https://stackoverflow.com/questions/44531510/why-not-always-use-the-index-as-the-key-in-a-vue-js-for-loop